source/getting-started/index.markdown

```diff
@@ -62,7 +62,7 @@ We will need a few things to get started with installing Home Assistant. The Ras
 5. Insert the SD card into your Raspberry Pi. If you are going to use an Ethernet cable, connect that too.
 6. Connect your power supply to the Raspberry Pi.
 7. The Raspberry Pi will now boot up, connect to the Internet and download the latest version of Home Assistant. This will take about 20 minutes.
-8. Home Assistant will be available at `http://hassio.local:8123`. If you are running an older Windows version or have a stricter network configuration, you might need to access Home Assistant at `http://hassio:8123`.
+8. Home Assistant will be available at `http://X.X.X.X:8123` (replace `X.X.X.X` with your Pi's IP address) or `http://hassio.local:8123`. If you are running an older Windows version or have a stricter network configuration, you might need to access Home Assistant at `http://hassio:8123`.
```

Comment:
I rather add to the end:

> Home Assistant will be available at `http://hassio.local:8123`. If you are running an older Windows version or have a stricter network configuration, you might need to access Home Assistant at `http://hassio:8123` or `http://X.X.X.X:8123` (replace `X.X.X.X` with your Pi's IP address).

The problem with your suggestion is, is that it is the first suggested, which indicates that it is normal, which it is not.

Reply:
Good point, edited.